### PR TITLE
abook: update to 0.6.1.

### PR DESCRIPTION
**Pull request by CasperVector**
Optional support for .vcf files was introduced in this version, but it [requires](https://gitweb.gentoo.org/repo/gentoo.git/tree/app-misc/abook/abook-0.6.1-r1.ebuild) [libvformat](https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-libs/libvformat/libvformat-1.13-r2.ebuild), and is therefore not enabled here.

### CODE REVIEW

**Comment by maxice8** [comment]
abook 0.6.1 is development version.

**Comment by CasperVector** [comment]
Indeed, but the version was released in 2015, and has been marked "stable" in 2017 on gentoo for all architetures it supports.

**Comment by maxice8** [comment]
On Thu, Mar 08, 2018 at 02:33:16PM +0000, Casper Ti. Vector wrote:
> Indeed, but the version was released in 2015, and has been marked "stable" in
> 2017 on gentoo for all architetures it supports.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.*
> 

Gentoo is not upstream in this case, but this should pass by a void member.

**Comment by CasperVector** [comment]
Yes, gentoo was not intended as an authoritative evidence, but as a (weakly?) supporting argument :)

**Comment by Vaelatern** [comment]
Considering the stalled upstream, if an abook user is okay with this upgrade I can accept it.

**Comment by CasperVector** [comment]
I can personally testify that this version always worked as expected on gentoo, and still works for me on void.  I will look into the CI failures on travis.

**Comment by CasperVector** [comment]
`/usr/share/gettext/archive.dir.tar.xz: No such file or directory`?
Not sure what happened to gettext-devel on travis for the affected architectures...

**Comment by Vaelatern** [comment]
Those are the cross builds.

**Comment by Vaelatern** [comment]
Thus /usr isn't in the normal spot?

**Comment by Vaelatern** [comment]
Or perhaps gettext-devel needs to be installed for the hostarch too?

**Comment by CasperVector** [comment]
I finally understood what you mean by reading `common/xbps-src/shutils/common.sh`...
gettext-devel moved from `$makedepends` to `$hostmakedepends`; cross build succeeded on my machine.

**Comment by Vaelatern** [comment]
so maybe `gettext` needs to be in `hostmakedepends` and not `gettext-devel`

**Comment by CasperVector** [comment]
`/usr/share/gettext/archive.dir.tar.xz` is provided by gettext-devel, not gettext...
(Just tried, the build would fail with gettext-devel replaced by gettext.)

File: srcpkgs/abook/template [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
-checksum=0646f6311a94ad3341812a4de12a5a940a7a44d5cb6e9da5b0930aae9f44756e
+distfiles="http://abook.sourceforge.net/devel/${pkgname}-${version}.tar.gz"
+checksum=f0a90df8694fb34685ecdd45d97db28b88046c15c95e7b0700596028bd8bc0f9
+
+post_extract() {
+	autoreconf --install
+}
+
+post_install() {
+	vdoc BUGS ChangeLog FAQ README TODO sample.abookrc
```
Comment (Vaelatern) [inline srcpkgs/abook/template]: This is invalid.
```
for f in BUGS ChangeLog FAQ README TODO sample.abookrc; do
vdoc $f
done
```

Comment (CasperVector) [reply srcpkgs/abook/template]: Fixed, plus the same for #12293.

**Pull request merged**